plugins/modules/zos_blockinfile.py

```diff
@@ -325,17 +325,18 @@
 """
 
 import json
+import traceback
 from ansible.module_utils.basic import AnsibleModule
 from ansible_collections.ibm.ibm_zos_core.plugins.module_utils import (
     better_arg_parser, data_set, backup as Backup)
 from ansible_collections.ibm.ibm_zos_core.plugins.module_utils.import_handler import (
-    MissingZOAUImport,
+    ZOAUImportError,
 )
 
 try:
     from zoautil_py import datasets
 except Exception:
-    Datasets = MissingZOAUImport()
+    Datasets = ZOAUImportError(traceback.format_exc())
```

Comment:
This one should be `datasets`, all in lower case, so the exception gets handled properly when calling the ZOAU API later in the code.